src/favouritesPage/components/favouriteCard.tsx

```diff
@@ -60,9 +60,31 @@ const FavouriteCard = (props: Props): JSX.Element => {
                 fullDesc={fullDescription}
               />
             ))}
+            {fullDescription && target.info_links.length > 0 && (
+              <Col className="FavouriteCard__links">
+                <h4>
+                  {t(
+                    'favouritesPage.targetCard.infoLinkHeader',
+                    'More information'
+                  )}
+                </h4>
+                {target.info_links.map((link, index) => {
+                  if (link.language === i18n.language) {
+                    return (
+                      <Fragment key={link.id}>
```

Comment:
Check out the implementation on the sidebar, where I've made these into an unordered list and then styled the default list styles away with CSS. That should be semantically more correct from the screen reader perspective.

Reply:
On other thought, that logic could probably just be separated into a new component that can then be used here as is.

Reply:
I can make a component from what you have done in single target view and use it here.